### PR TITLE
Feature/step2 modeling actions

### PR DESCRIPTION
**Pull request by kravets-levko**

### CODE REVIEW

File: app/services/package.js [hunk 1]
```diff
@@ -7,85 +7,112 @@ var registry = require('datapackage-registry');
...
-    return new Promise(function(resolve, reject) {
-      utils.getCsvSchema(urlOrFile)
+module.exports.getFiscalDataPackageSchema = function() {
+  return module.exports.getDataPackageSchema('fiscal');
+};
+
+module.exports.getDataPackageSchema = function(schemaId) {
+  return new Promise(function(resolve, reject) {
+    var options = {
+      backend: '/proxy?url=' + encodeURIComponent('https://rawgit.com/' +
```
Comment (pwalsh) [inline app/services/package.js]: - FYI this is now the offical location for the schema registry: http://schemas.datapackages.org/registry.csv

**Comment by pwalsh** [comment]
@kravets-levko can you change the shippable file to a travis file. I want to move these few repos off shippable and have all our stuff on travis by default. Also - tests are failing.

**Comment by pwalsh** [comment]
Looks like the fail is just that `istanbul` can't be found on shippable. The travis fail is just because there is no Travis file, so it defaults to thinking it is a Ruby codebase because... yeah, Travis.

**Comment by kravets-levko** [comment]
@pwalsh Changed url to datapackage registry; added travis.yml. Tests are successfully running on my PC and (as I see) on shippable. Shippable fails because it cannot push sources to heroku - this is allowed only from openspending's master

**Comment by pwalsh** [comment]
@kravets-levko ok that is fine. Let's remove shippable completely and you can just push manually to heroku. We'll update our continuous deployment stuff in Jan ( cc @roll ).

**Pull request merged**